nixos/modules/services/ttys/getty.nix

```diff
@@ -7,14 +7,26 @@ let
 
   baseArgs = [
     "--login-program" "${cfg.loginProgram}"
-  ] ++ optionals (cfg.autologinUser != null) [
+  ] ++ optionals (cfg.autologinUser != null && !cfg.autologinOnce) [
     "--autologin" cfg.autologinUser
   ] ++ optionals (cfg.loginOptions != null) [
     "--login-options" cfg.loginOptions
   ] ++ cfg.extraArgs;
 
   gettyCmd = args:
-    "@${pkgs.util-linux}/sbin/agetty agetty ${escapeShellArgs baseArgs} ${args}";
+    "${pkgs.util-linux}/sbin/agetty ${escapeShellArgs baseArgs} ${args}";
```

Comment:
I removed this thing to change the argv[0] because it wouldn't allow me to reuse gettyCmd in the script.
I don't understand why it be required and removing it didn't (apparently) break anything.

Reply:
@rnhmjoj, I think the `argv[0]` thing could be there to make the command name look cleaner in `systemctl status`, `top`, etc.